content/en/synthetics/browser_tests/actions.md

```diff
@@ -128,88 +161,98 @@ await promise
-If a test contains an assertion on the response body and the timeout limit is reached, an `Assertions on the body/response cannot be run beyond this limit` error appears.
+[1]: /synthetics/guide/email-validation
+[2]: /synthetics/browser_tests/actions#use-variables
+[3]: /synthetics/guide/testing-file-upload-and-download/#testing-a-file-download
+{{% /tab %}}
+{{< /tabs >}}
 
 ### Navigation
 
-{{< img src="synthetics/browser_tests/navigation_step.png" alt="Browser Test Navigation Step" style="width:60%;">}}
+{{< img src="synthetics/browser_tests/navigation_step.png" alt="Choose between three navigation types in a browser test recording" style="width:60%;" >}}
 
 #### Refresh a page
 
-Refresh the current page of the scenario.
+Create this navigation step to have your browser test refresh the current page of the recording.
 
 #### Go to an email and click on a link
 
-This step allows you to access unique Synthetic mail inboxes after creating an [email variable][4]. Choose the email you are interested in and click the link you want your browser test to click on. This step brings your test to the corresponding page and allows you to move on with the rest of your journey from that specific page.
+Once you have [created an email variable](#test-that-an-email-was-received), create this navigation step to have your browser test access unique Synthetic mail inboxes.
 
-#### Follow a specific link
+Select the email and links you want the browser test to click on. This step brings you to the corresponding page and allows you to move on with the rest of your journey from that specific page.
 
-Go to a specific page.
+#### Follow a specific link
 
-**Note**: You must prepend your URLs with `http` or `https` in the **Enter link URL** box.
+Create this navigation step to have your browser test go to a specific page. You must prepend your URLs with `http` or `https` in the **Enter link URL** box.
 
 ### Special actions
 
-The [Datadog browser test recorder extension][3] is able to record most steps associated to user journeys you might want to monitor. However, some steps—such as **Hover**, **Press Key**, and **Scroll**—are not recorded automatically. Explicitly add a step for them using the **Special Actions** menu located at the top left-hand corner of the recorder.
+You can use the [Datadog browser test recorder extension][3] to record and monitor most steps associated with user journeys. However, some steps such as **Hover**, **Press Key**, **Scroll**, and **Wait** are not recorded automatically. 
+
+Create this assertion step manually by clicking **Special Actions** and selecting an action type.
 
 #### Hover
 
-To avoid generating a step at each element hovering during recording, this browser test step is not added through an actual hovering mechanism but using a dedicated step with a click.
+To avoid generating a step for every time a user hovers over an element during recording, this step uses a dedicated step with a click, not a hovering mechanism.
```

Comment:
```suggestion
This step uses a dedicated click, not a hovering mechanism, to avoid generating a separate step every time a user hovers over an element during recording.
```